File: lib/player.rb
class Player
  attr_reader :id, :socket

  def initialize(id, socket)
    @id = id
    @socket = socket
  end

  def send(message)
    @socket.send(message)
  end

  def kick
    @socket.send('Your connection has been closed.')
    @socket.close
  end
end

File: lib/player.js
class Player {
  constructor(id, socket) {
    this.id = id;
    this.socket = socket;
  }

  send(message) {
    this.socket.send(message);
  }

  kick() {
    this.socket.send('Your connection has been closed.');
    this.socket.close();
  }
}

module.exports = Player;
